storyblok/storyblok-design-system
# Feat/int 282 datepicker

This pull request adds a flag to deliver the date as Iso Date. If set to true, the date would be set in content object as Iso date. 

## Pull request type

Jira Link: [INT-282](https://storyblok.atlassian.net/browse/INT-282)




## How to test this PR


Just use the Datepicker and set the prop flag iso-date as true. After saving the story, the content object must record the Datepicker data as Iso Date. If set to false, the value should be recorded normaly.

## What is the new behavior?



- A new flag was added to set the date as Iso Date. No further configuration or breaking changes are expected to this feature to work properly.

## Other information
